### PR TITLE
This PR refers to Issue #6, i.e to create a Schema for education history of the students.

### PR DESCRIPTION
**Pull request by Shivam-53**
I  have created the schema for student education details, also i have used local mongodb

### CODE REVIEW

File: models/databaseUtil.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const mongoose = require('mongoose');
 require("dotenv").config();
 
 mongoose.set('strictQuery', true);
+// let DB_URL="mongodb://localhost:27017/new_data"
```
Comment (Hitansh159) [inline models/databaseUtil.js:5]: remove this comment

File: routes/auth.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ const authenticateToken = require('../middleware/auth');
 
 router.post('/', authController.login);
 
+
```
Comment (Hitansh159) [inline routes/auth.js:9]: remove empty line

File: models/std_edu_history.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
+import mongoose from 'mongoose';
+const { Schema, default: mongoose } = require('mongoose');
```
Comment (Hitansh159) [inline models/std_edu_history.js:2]: no need of this import

**Review by Hitansh159 — CHANGES_REQUESTED** [review]

**Comment by Hitansh159** [comment]
Also mention which Issue this PR refers to

**Comment by Shivam-53** [comment]
Is everything good now?

**Review by Hitansh159 — APPROVED** [review]

**Pull request merged**